main/cryptsetup/APKBUILD

```diff
@@ -43,7 +49,14 @@ package() {
 	install -Dm755 "$srcdir"/dmcrypt.initd "$pkgdir"/etc/init.d/dmcrypt
 
 	mkdir -p "$pkgdir"/usr/share/doc/$pkgname/
-	install -m644 README TODO FAQ "$pkgdir"/usr/share/doc/$pkgname/
+	install -m644 README TODO FAQ NEWS docs/v${pkgver}-ReleaseNotes \
+		"$pkgdir"/usr/share/doc/$pkgname/
+
+	# From v2.0.0-ReleaseNotes:
+	#   If your distro does not support tmpfiles.d directory, you
+	#   have to create locking directory (/run/lock/cryptsetup) in
+	#   cryptsetup package (or init scripts).
+	mkdir -p "$pkgdir"/"$_lockdir"
```

Comment:
`/run` is tmpfs, it’s wiped on each restart, so it does not make sense and should not be created by the package, but runscript.

Reply:
Ah, indeed it is actually not even necessary to create the directory in the package since it will in fact be created by cryptsetup. It will just emit a warning when creating the directory (see https://gitlab.com/cryptsetup/cryptsetup/issues/362).

I would suggest simply not creating the directory.

Reply:
I removed the creation of the locking directory from the APKBUILD.